src/handleErrors.js

```diff
@@ -9,35 +9,52 @@ export function printNoMockFoundError(
   operationName: string,
   variables: Variables
 ): void {
-  const mockedQueries = Object.keys(queryMock._queries);
-  throw new Error(
-    `No suitable mock for operation "${operationName ||
-      'unknown'}" with variables ${JSON.stringify(
-      variables
-    )} found. Please make sure you have mocked the query you are making.${
-      mockedQueries.length > 0
-        ? '\n\n === Currently mocked queries ===\n' +
-          mockedQueries
-            .map(
-              queryName =>
-                `"${queryName}" with variables: \n\n  ${queryMock._queries[
-                  queryName
-                ]
-                  .map(
-                    ({ queryMockConfig }) =>
-                      `${JSON.stringify(
-                        queryMockConfig.variables
-                      )}, diff: \n${printDiff(
-                        variables,
-                        queryMockConfig.variables
-                      )}`
-                  )
-                  .join('\n\n  ')}`
-            )
-            .join(', ')
-        : ''
-    }`
+  const errorMessageArray = getNoMockFoundErrorMessageArray(queryMock, operationName, variables);
+  throw new Error(errorMessageArray.join(''));
+}
+
+function getNoMockFoundErrorMessageArray(
+  queryMock: QueryMock,
+  operationName: string,
+  variables: Variables
+) : Array<string> {
+  const mockedQueriesMessageArray = [
+    `Could not find matching mock for operation "${operationName}" ` +
+    `with variables: ${JSON.stringify(variables)}\n` +
+    `Make sure you have mocked the query you are making.\n\n`
+  ];
+
+  if (!Object.entries(queryMock._queries).length) {
+    mockedQueriesMessageArray.push(`=== No query was mocked ===\n\n`);
+    return mockedQueriesMessageArray;
+  }
+
+  if (queryMock._queries[operationName]) {
+    mockedQueriesMessageArray.push(`=== Currently mocked "${operationName}" queries ===\n`);
+    queryMock._queries[operationName].forEach(({queryMockConfig}) => {
+        mockedQueriesMessageArray.push(
+          `Query "variables": ${JSON.stringify(queryMockConfig.variables)}\n` +
+          `Diff of "variables":\n` +
+          `${printDiff(queryMockConfig.variables, variables)}\n\n`
+        );
+    });
+  }
+
+  const otherMockedQueryNames = Object.keys(queryMock._queries).filter(
+    (queryMockName) => (operationName != queryMockName)
```

Comment:
Mind making this a strict equality check? `!==`.

Reply:
:thumbsup: